packages/ember-runtime/lib/mixins/controller.js

```diff
@@ -20,11 +20,10 @@ export default Mixin.create(ActionHandler, ControllerContentModelAliasDeprecatio
     it will attempt to send the action to the view's controller's `target`.
 
     By default, the value of the target property is set to the router, and
-    is injected when a controller is instantiated. This injection is defined
-    in Ember.Application#buildContainer, and is applied as part of the
-    applications initialization process. It can also be set after a controller
-    has been instantiated, for instance when using the render helper in a
-    template, or when a controller is used as an `itemController`. In most
+    is injected when a controller is instantiated. This injection is applied as
+    part of the applications initialization process. It can also be set after a
+    controller has been instantiated, for instance when using the render helper
+    in a template, or when a controller is used as an `itemController`. In most
```

Comment:
@rwjblue I removed the reference to `itemController` and the `render` helper. I didn't see the `render` template mentioned in the template guides, seems not really relevant anyway.